data/simpy.gram

```diff
@@ -114,90 +110,90 @@ dotted_name[expr_ty]:
     | NAME
 
 if_stmt[stmt_ty]:
-    | a='if' b=named_expression ':' c=block d=else_block {
-        _Py_If(b, c, d, EXTRA) }
-    | a='if' b=named_expression ':' c=block e=elif_stmt {
-        _Py_If(b, c, singleton_seq(p, e), EXTRA) }
-    | a='if' b=named_expression ':' c=block {
-        _Py_If(b, c, NULL, EXTRA) }
+    | 'if' a=named_expression ':' b=block c=else_block {
+        _Py_If(a, b, c, EXTRA) }
+    | 'if' a=named_expression ':' b=block c=elif_stmt {
+        _Py_If(a, b, singleton_seq(p, c), EXTRA) }
+    | 'if' a=named_expression ':' b=block {
+        _Py_If(a, b, NULL, EXTRA) }
```

Comment:
The first and 3rd alternatives (if-else and if without anything) can be combined into one using `c=[else_block]`. (The elif variant should go first to deal with the ambiguity.)